tests/framework/session/test_session.py

```diff
@@ -85,34 +92,41 @@ def mock_config_loader_class(mocker):
 @pytest.fixture
 def mock_settings_config_loader_class(mocker, mock_config_loader_class):
     class MockSettings(_ProjectSettings):
-        _CONFIG_LOADER_CLASS = Validator(
+        mocker.patch("kedro.framework.project.issubclass")
```

Comment:
This line surprised me and got a bit confused in the tests - do you remember why we need to mock the config loader class instead of using `ConfigLoader` as is? (L88-89)

Reply:
Very good catch! 🦅   It seems like it was mocked to make `_get_logging_config` work. I've now just mocked the call to `_get_logging_config` instead and removed the mocked config loader.

Reply:
Nice thank you! Do we need to do the same in `test_settings.py` as well?

Reply:
Done!